package.json

```diff
@@ -23,7 +23,7 @@
     "q": "~1.0.0",
     "request": "~2.33.0",
     "selenium-webdriver": "~2.41.0",
-    "scribe-test-harness": "~0.0.7"
+    "scribe-test-harness": "~0.0.14"
```

Comment:
This is just a placeholder version number as I have a separate PR out for the harness that would need to be released first.